src/core_plugins/inspector_views/public/requests/requests_view.js

```diff
@@ -92,6 +96,25 @@ class RequestsViewComponent extends Component {
       req => req.status === RequestStatus.ERROR
     ).length;
 
+    let showInConsole;
+    if (this.state.request
+      && this.state.request.isEsSearchRequest
+      && this.state.request.esIndex
+      && this.state.request.json) {
+      const searchRequest = rison.encode(this.state.request.json);
+      showInConsole = (
+        <Fragment>
+          <EuiSpacer size="xs"/>
+          <EuiButton
+            href={chrome.addBasePath(`/app/kibana#/dev_tools/console?search_request=${searchRequest}&index=${this.state.request.esIndex}`)}
```

Comment:
You may need to also encode `this.state.request.esIndex`. IIRC, an ES index name can have characters that are not URL-safe (like `&`)

Reply:
Yup this is true